src/Compilers/CSharp/Portable/Symbols/Source/SourceConstructorSymbol.cs

```diff
@@ -12,7 +12,6 @@ namespace Microsoft.CodeAnalysis.CSharp.Symbols
 {
     internal sealed class SourceConstructorSymbol : SourceConstructorSymbolBase
     {
-        private readonly bool _isExpressionBodied;
         private readonly bool _hasThisInitializer;
```

Comment:
we could consider moving this bit down as well.  That would save another unnecessary 32bits in constructors.  however, constructors seem to be orders of magnitude less common than methods, so it's not high on my priority list.